rollingpin/deploy.py

```diff
@@ -82,7 +82,7 @@ def process_host(self, host, commands, timeout=0):
                 yield self.event_bus.trigger(
                     "host.command", host=host, command=command)
                 result = yield connection.execute(log, command, timeout)
-                results.append(result)
+                results.append((command, result))
```

Comment:
Yeah I agree, that was confusing for me and I was working on it directly!  I like the namedtuple idea.